wiki/Music_theory/Layer/en.md

```diff
@@ -0,0 +1,25 @@
+---
+tags:
+  - musical layer
+  - sound layer
+---
+
+# Layer
+
+A **musical layer** is a part of the song which contains features with a common aspect between them. A significant part of [beatmapping](/wiki/Beatmapping) in every [game mode](/wiki/Game_mode) revolves around consistently following different kinds of layers (such as percussion, vocals, and sound patterns), and switching between them for the purpose of emphasis and contrast.
+
+## Description
+
```

Comment:
correct me if i'm wrong, but this kind of section isn't really a thing in other articles

```suggestion
```

Reply:
in the music theory? yes, it isn't. my intention was to make a very brier description of what a layer is, then expand on every side or that in separate sections. think wikipedia, a well-formed article has a leading section with a short overview, followed by a detailed breakdown. it usually works better than large pieces of text without any subdivision, especially in large articles, so I wanted to give it a go for smaller topics as well

Reply:
on a second thought, the page the way I wrote it looks better without this heading since the second paragraph reads smoothly enough as a continuation of the first one